test/Microsoft.Health.Dicom.Tests.Integration/Persistence/CustomTagStoreTests.STOW.cs

```diff
@@ -0,0 +1,180 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Dicom;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+using Microsoft.Health.Dicom.SqlServer.Features.CustomTag;
+using Microsoft.Health.Dicom.Tests.Common;
+using Microsoft.Health.Dicom.Tests.Common.Extensions;
+using Microsoft.Health.Dicom.Tests.Integration.Persistence.Models;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Tests.Integration.Persistence
+{
```

Comment:
Can we name this file differently? It isn't exactly testing the CustomTagStore directly, it more tests the index data store. So perhaps IndexDataStoreTests.CustomTag.cs?

Reply:
sure